### ISSUE TITLE
Doesnt seem to work after buildStart

### ISSUE THREAD

**Issue by jakewhiteley** [ModyQyW/vite-plugin-stylelint#39]
### Describe the bug

I load up a simple app with a stylelint error (in this case I have added `0px` as a value, so the ` Unexpected unit ` rule is triggered).
Saving the file after a change witht he error still there shows no console error. Adding a new styleint error into the file does not report any error.

On first load, I see an error in my console. If I create any other stylelint error, it will not output in my console. 

If I start vite with no errors and then introduce one, it does not report any error.

Perhaps the issue is that I do not include any styles in my entry point at all?

I treat my scss as a seperate input:
```
build: {
	manifest: true,
	emptyOutDir: true,
	outDir: 'public',
	assetsDir: 'build',
	rollupOptions: {
		input: {
			js: `${env.VITE_SRC_DIR}/js/theme.js`,
			styles: `${env.VITE_SRC_DIR}/sass/style.scss`
		}
	}
},
```

### Reproduction

This is my styleint config

```
{
    "extends": "stylelint-config-sass-guidelines",
    "rules": {
        "indentation": "tab",
        "string-quotes": null,
        "number-leading-zero": null,
        "no-missing-end-of-source-newline": null,
        "order/properties-alphabetical-order": null,
        "property-no-vendor-prefix": null,
        "max-nesting-depth": null,
        "selector-max-compound-selectors": 5,
        "selector-class-pattern": null,
        "selector-pseudo-element-colon-notation": "single",
        "declaration-property-value-disallowed-list": null,
        "color-named": [
            "never",
            {
                "ignore": ["inside-function"]
            }
        ],
        "selector-no-qualifying-type": [
            true,
            {
                "ignore": [
                    "attribute"
                ]
            }
        ]
    }
}
```

this is my plugin config:
```
stylelint({
	fix: false,
	include: ``${env.VITE_SRC_DIR}/sass/**/*.scss`,
	lintOnStart: false,
	lintInWorker: true
	// emitErrorAsWarning: true
})
```

And this is an offending scss file:

```
pre {
	margin-top: 0px;
	// top: 0px;
}
```

### System Info

```Shell
System:
    OS: Windows 10 10.0.22621
    CPU: (20) x64 12th Gen Intel(R) Core(TM) i7-12700KF
    Memory: 40.96 GB / 63.86 GB
  Binaries:
    Node: 16.14.0 - C:\Program Files\nodejs\node.EXE
    npm: 8.3.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.963.0), Chromium (108.0.1462.76)
    Internet Explorer: 11.0.22621.1
```
```


### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ModyQyW/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by jakewhiteley** [comment]
Found it: `chokidar` needed to be true.

**Comment by ModyQyW** [comment]
I think this problem is listed in FAQ as `cache is broken`. Feel free to send a PR if you have a good solution. Thanks.

<img width="855" alt="image" src="https://user-images.githubusercontent.com/31850793/211697629-590a8c42-f0f9-495f-b54c-7eddc0412902.png">
